docker/gaffer-pyspark-notebook/.env

```diff
@@ -4,6 +4,7 @@ GAFFER_TOOLS_VERSION=1.21.1
 ACCUMULO_VERSION=1.9.3
 HADOOP_VERSION=3.2.2
 SPARK_VERSION=3.1.2
+KUBECTL_VERSION=1.23.0
```

Comment:
Do we want to manage this? It seems like GitHub Actions runners have a different [version](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md)?

Reply:
So this manages the version of kubectl that we install inside the notebook container image. It doesn't change anything to do with the version of kubectl that gets installed on the github runner used to run the tests.

All my change does is pull the default version that gets installed into the notebook out of Dockerfile into .env so that I can pick it up and use it in update_versions.sh to update the notebook description.

Reply:
Ah yes of course I forgot it gets installed into the notebook container.